include/mbedtls/config.h

```diff
@@ -3671,14 +3671,6 @@
  */
 //#define MBEDTLS_PSA_HMAC_DRBG_MD_TYPE MBEDTLS_MD_SHA256
 
-/** \def MBEDTLS_PSA_MAX_KEY_BITS
- * Restrict the PSA Library to supporting a maximum key size (in bits).
- *
- * If this option is unset, the library will fall back to a default value of
- * 8191 bytes = 65528 bits.
- */
-//#define MBEDTLS_PSA_MAX_KEY_BITS 0xfff8
-
```

Comment:
Please use meaningful commit messages. “Apply review feedback” is not meaningful. This commit should be broken up into meaningful pieces (here, two: moving the config.h entry, and adding the static assertion).

Reply:
Done (getting rid of MAX_KEY_BITS removed the commit)